server/data/manageAdjudicationsClient.ts

```diff
@@ -52,4 +57,23 @@ export default class ManageAdjudicationsClient {
       path: `/reported-adjudications/${adjudicationNumber}`,
     })
   }
+
+  async getCompletedAdjudications(
+    user: User,
+    pageRequest: PageRequest
+  ): Promise<PageResponse<CompletedAdjudicationSummary>> {
+    return pageResponseFrom(pageRequest, dummyResults)
+  }
 }
+
+const dummyResults: CompletedAdjudicationSummary[] = generateRange(1, 1, _ => {
```

Comment:
Should this live in a test helper module of some sort?

Reply:
Yeah - I can pull it out somewhere else - its basically to be purged when I'm done - the only reason its here is so that I can test what the front end looks like before the back end apis are in place